ios/FluentUI.Demo/FluentUI.Demo/Demos/ActivityIndicatorDemoController.swift

```diff
@@ -233,8 +235,9 @@ class ActivityIndicatorDemoController: DemoTableViewController {
 
     private let xLargeSize: CGFloat = 36
 
-    @objc private func startStopActivity() {
+    @objc private func startStopActivity(_ sender: UIButton) {
         isAnimating.toggle()
+        sender.setTitle(activityActionButtonTitle, for: .normal)
```

Comment:
Instead of coupling the UI type of the sender to this logic, it might be safer to reload the contents of the relevant table view cell and let the existing logic do the work.

Reply:
Yeah, sounds good. 

Code updated to reload row on button action.